frontend/src/metabase/dashboard/components/DashboardHistoryModal.jsx

```diff
@@ -4,24 +4,37 @@ import { withRouter } from "react-router";
 
 import HistoryModal from "metabase/containers/HistoryModal";
 import * as Urls from "metabase/lib/urls";
+import * as dashboardActions from "metabase/dashboard/actions";
 import Dashboards from "metabase/entities/dashboards";
+import connect from "react-redux/lib/connect/connect";
 
 @withRouter
 @Dashboards.load({
   id: (state, props) => Urls.extractEntityId(props.params.slug),
   wrapped: false,
 })
+@connect(
+  null,
+  dashboardActions,
+)
```

Comment:
Mind binding only used actions? Binding everything leads to [prop drilling](https://kentcdodds.com/blog/prop-drilling). This can be changed to:

```jsx
import { fetchDashboard, fetchDashboardCardData } from "metabase/dashboard/actions"

const mapDispatchToProps = {
   fetchDashboard,
   fetchDashboardCardData,
}

@connect(
   null,
   mapDispatchToProps,
)
export default class ...
```